scripts/zones/Phanauet_Channel/npcs/Ineuteniace.lua

```diff
@@ -2,15 +2,26 @@
 -- Area: Phanauet Channel
 --  NPC: Ineuteniace
 -- Type: Standard NPC
+-- Time: Central to South via Newtpool (Different speaking text)
+-- Time: South to North via main canal (Different speaking text)
 -- !pos 11.701 -3 1.360 1
+-- NOTES/TODO: Implement NPCs on boat and dialog
+-- Main Canal
+-- Message 7355 10 seconds after loading
+-- Message 7356 50 seconds later
+-- Message 7357 50 seconds later
+-- Message 7361 after Fishing Animation 2 min later
+-- Message 7370 when talking to him (on both)
+-- Message 7358
+-- Message 7360 Arriving message then you arrive (about 30 seconds before)
 -----------------------------------
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    player:startEvent(101)
+    player:startEvent(101) -- Not sure what this event is
```

Comment:
Going by Velyn's event site, it seems SE may have had some code to trigger what he says depending on the time.

```
### "Ineuteniace" (16781345)
EventEntryPoint_16781345_65535.L0000:
.L0000: END

EventEntryPoint_16781345_101.L0001:
.L0001: NPCSAY #7334 // Time: {e:0}\x07Route: {e:1}<7F31>
        WAITFORINPUT
        FINISHEVENT
        END
```